api/reporting/docs/cohort_definition_schema.json

```diff
@@ -0,0 +1,201 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Confirmed the schema is  valid and a JSON  entry from the DB matches it using https://www.jsonschemavalidator.net/. Good stuff.